implicit ambiguities in SIP-18 warnings are reported without position

```scala
    object Test {
      implicit lazy val ambiguousPostfixOps1: language.postfixOps.type
= language.postfixOps
      implicit lazy val ambiguousPostfixOps2: language.postfixOps.type
= language.postfixOps
      List(1, 2, 3) tail
    }
```

Compiling gives us:

```
    error: ambiguous implicit values:
     both value ambiguousPostfixOps1 in object Test of type =>
language.postfixOps.type
     and value ambiguousPostfixOps2 in object Test of type =>
language.postfixOps.type
     match expected type scala.languageFeature.postfixOps
    one error found
```

```
➜  snips git:(master) ✗ ~/scala-2.10.7/bin/scalac -d /tmp t8176.scala
error: ambiguous implicit values:
 both value ambiguousPostfixOps1 in object Test of type => language.postfixOps.type
 and value ambiguousPostfixOps2 in object Test of type => language.postfixOps.type
 match expected type scala.languageFeature.postfixOps
one error found
➜  snips git:(master) ✗ scalac -d /tmp t8176.scala                   
t8176.scala:6: error: ambiguous implicit values:
 both lazy value ambiguousPostfixOps1 in object Test of type => language.postfixOps.type
 and lazy value ambiguousPostfixOps2 in object Test of type => language.postfixOps.type
 match expected type scala.languageFeature.postfixOps
      List(1, 2, 3) tail
                    ^
one error found
```

--- a/src/compiler/scala/tools/nsc/typechecker/Implicits.scala
+++ b/src/compiler/scala/tools/nsc/typechecker/Implicits.scala
@@ -1239,7 +1239,9 @@ trait Implicits {
               val pendingImprovingBest = undoLog undo {
                 otherPending filterNot firstPendingImproves
               }
-              rankImplicits(pendingImprovingBest, (newBest, firstPending) :: acc)
+
+              if (pt.typeSymbol.hasAnnotation(definitions.LanguageFeatureAnnot)) (newBest, firstPending):: Nil
+              else rankImplicits(pendingImprovingBest, (newBest, firstPending) :: acc)
           }
       }
 

--- a/test/files/pos/t8176.scala
+++ b/test/files/pos/t8176.scala
@@ -1,0 +1,5 @@
+object Test {
+  implicit lazy val ambiguousPostfixOps1: language.postfixOps.type = language.postfixOps
+  implicit lazy val ambiguousPostfixOps2: language.postfixOps.type = language.postfixOps
+  List(1, 2, 3) tail
+}

